config/routes.rb

```diff
@@ -36,4 +36,16 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  get "/apps", to: "apps#index"
+  get "apps/new", to: "apps#new"
+  get "/apps/search", to: "apps#show"
+  get "apps/:id", to: "apps#show"
+  post "/apps", to: "apps#create"
+  patch "/apps/:id", to: "apps#update"
```

Comment:
See notes in view and controllers about more RESTful approaches to these routes! These are things we're all still learning so it's very normal to get this type of feedback on this project!